wbstack/src/Settings/LocalSettings.php

```diff
@@ -568,9 +568,24 @@ function onPersonalUrlsConfirmAccount( array &$personal_urls, Title $title, Skin
     wfLoadExtension( 'CirrusSearch' );
     wfLoadExtension( 'WikibaseCirrusSearch' );
 
+    $wgWBRepoSettings['searchIndexTypes'] = [
+        'string', 'external-id', 'url', 'wikibase-item', 'wikibase-property'
+    ];
+
     // If Wikibase Lexemes are enabled, enable lexeme cirrus search
     if ( $wikiInfo->getSetting('wwExtEnableWikibaseLexeme') ) {
         wfLoadExtension('WikibaseLexemeCirrusSearch');
+
+        $wgLexemeUseCirrus = true;
+
+        // So that Lexemes are indexed in the content index
+        $wgContentNamespaces[] = 146;
+
+        // Add lexeme searchIndexTypes
+        $wgWBRepoSettings['searchIndexTypes'] = [
+            'string', 'external-id', 'url', 'wikibase-item', 'wikibase-property',
+            'wikibase-lexeme', 'wikibase-form', 'wikibase-sense'
+        ];
```

Comment:
Suggestion to avoid repeating the non-lexeme types:

```suggestion
        $wgWBRepoSettings['searchIndexTypes'][] = 'wikibase-lexeme';
        $wgWBRepoSettings['searchIndexTypes'][] = 'wikibase-form';
        $wgWBRepoSettings['searchIndexTypes'][] = 'wikibase-sense';
```

Reply:
good idea :+1: